api_tests/src/image.spec.ts

```diff
@@ -303,23 +299,17 @@ test("Create an image post, and make sure a custom thumbnail doesnt overwrite it
 
   const community = await createCommunity(alphaImage);
 
-  let post = await createPost(
+  const post = await createPost(
     alphaImage,
     community.community_view.community.id,
     upload1.url,
+    "https://example.com/",
+    randomString(10),
+    randomString(5),
+    upload2.url!
   );
   expect(post.post_view.post.url).toBe(upload1.url);
-
-  // Edit the post
-  await editPostThumbnail(alphaImage, post.post_view.post, upload2.url!);
-
-  // Wait for the metadata to get fetched
-  post = await waitUntil(
-    () => getPost(alphaImage, post.post_view.post.id),
-    p => p.post_view.post.thumbnail_url == upload1.url,
-  );
```

Comment:
I dont think this was working as expected, as the post already had this thumbnail url since it was created, it wouldnt wait for the edit at all but would return immediately. Thats why I moved the custom thumbnail param directly to createPost().